src/wp-includes/class-wp-block.php

```diff
@@ -235,14 +235,7 @@ private function process_block_bindings() {
 
 		$computed_attributes = array();
 
-		// Allowed blocks that support block bindings.
-		// TODO: Look for a mechanism to opt-in for this. Maybe adding a property to block attributes?
-		$allowed_blocks = array(
-			'core/paragraph' => array( 'content' ),
-			'core/heading'   => array( 'content' ),
-			'core/image'     => array( 'url', 'title', 'alt' ),
-			'core/button'    => array( 'url', 'text', 'linkTarget', 'rel' ),
-		);
+		$allowed_blocks = WP_Block_Bindings_Registry::get_instance()->allowed_blocks;
```

Comment:
That part leaks the temporary list of allowed blocks so it would become a public API. We need to ensure it doesn't happen and find an alternative approach.

Reply:
You're right. We can go back to the hardcoded variable.

Reply:
I went back to the hardcoded variable: [link](https://github.com/WordPress/wordpress-develop/pull/6079/files#diff-c569b1ecb11a007a563788a481710b51e0cb21cd1d1885490bf9c91777842f23R238-R243).